### ISSUE TITLE
#Admin?sub=servers - uses IP instead of FQDN when SSL enabled (POST /api/app/add_server )

### ISSUE THREAD

**Issue by care2DavidDPD** [jhuckaby/Cronicle#258]
## Summary
When POST `/api/app/add_server` is called from the "add server" button, the IP address is passed instead of the FQDN/hostname.  When using verified SSL Certificates, the IP is not in the certificate.

Clicking " add server ... " and typing the FQDN, does match with the discovered entry in the list and works just fine.  

## Steps to reproduce the problem
Use SSL with verified certificates from Let's Encrypt.

### Cronicle software version?
0.8.39

### Recommend solution 
Use the FQDN, or pass both and catch the exception on ERR_TLS_CERT_ALTNAME_INVALID - and retry with hostname.

**Comment by jhuckaby** [comment]
Yeah, Cronicle has a huge number of bugs with HTTPS support.  This is definitely one of them, but it's going to take a large overhaul of a bunch of things to fully support SSL between servers.
